UN-OCHA/hpc-content-site
# release versions/v3.4.1

- chore: Update all outdated drupal/* packages.
- Fix some issues before updating
- chore: Bump the docker image to PHP 8.2 for the new nginx aggregation rules.
- fix: Ensure the required libraries are at versions that are happy with PHP 8.2 and also set this as required in composer.json.
- fix: Remove preserve-paths in favour of the new solution, which means no need anymore to pin composer.
- Update to drush 12 and updated the post-deploy docksal command to use the drush deploy command
- Deprecation warning fix also in settings file used for tests
- Fixing composer issues
- Updated composer.json with changes after composer install
- fix: Always allow inclusion of extra settings, so the new sanity test does not fail.
- Update config after latest updates of Drupal core
- chore: Update all outdated drupal/* packages.
- chore: Update all outdated drupal/* packages.
- Remove patch to make updates work
- HPC-9156: Add and enable the Username Enumeration module
- chore: Update all outdated drupal/* packages.
- chore: Update all outdated drupal/* packages.
- Check field before setting a value
